vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -274,6 +274,19 @@ type ExecStartCheck struct {
 	Tty bool
 }
 
+type HealthcheckResult struct {
+	Start, End time.Time
+	ExitCode   int    // 0=healthy, 1=unhealthy, 2=starting
+	Output     string // Output from last check
```

Comment:
Yes, but my understanding is that Go's json encoder will escape any values that are not valid unicode.

The problem arises when you try to decode a JSON string that's not properly escaped. But Go should apply the correct escaping rules.